src/ServiceControlInstaller.Engine/Instances/MonitoringInstance.cs

```diff
@@ -188,13 +188,9 @@ public void ApplyConfigChange()
             {
                 try
                 {
-                    QueueCreation.RunQueueCreation(this);
+                    InstanceSetup.Run(this);
                 }
-                catch (QueueCreationFailedException ex)
```

Comment:
We had no logic that needed explicit exception classes to be used so removed them. This now also catches plain `Exception` which previously would have crashed the installer